dspace-api/src/main/java/org/dspace/content/Collection.java

```diff
@@ -202,31 +188,16 @@ void setAdmins(Group admins) {
         setModified();
     }
 
-    public Group getWorkflowStep1() {
-        return workflowStep1;
-    }
-
-    public Group getWorkflowStep2() {
-        return workflowStep2;
+    public Group getWorkflowStep1(Context context) {
+        return getCollectionService().getWorkflowGroup(context, this, 1);
     }
 
-    public Group getWorkflowStep3() {
-        return workflowStep3;
+    public Group getWorkflowStep2(Context context) {
```

Comment:
this is out the scope of this PR as stated in the PR description as it will require more work
> complete removal of the Basic Workflow (it will be done in one or multiple PRs, see for example #2268)

Reply:
see https://github.com/DSpace/DSpace/pull/2312#discussion_r263173554